utilities/constants/constants.go

```diff
@@ -10,6 +10,7 @@ const PullImageLabels = "io.rancher.container.pull_image"
 const UUIDLabel = "io.rancher.container.uuid"
 const CattelURLLabel = "io.rancher.container.cattle_url"
 const AgentIDLabel = "io.rancher.container.agent_id"
+const RancherAgentImage = "io.rancher.host.rancher_agent_image"
```

Comment:
@StrongMonkey can you change this to `io.rancher.host.agent_image`.  We prefix the env vars with `RANCHER_` to namespace them, but the label is already namespaced to `io.rancher...`